OneMore/Models/Page.cs

```diff
@@ -277,6 +275,55 @@ public void AddTagDef(TagDef tagdef)
 			Root.AddFirst(tagdef);
 		}
 
+		/// <summary>
+		/// Extended version from RemindCommand to handle also non Todo Tags
+		/// </summary>
+		public string SetTag(XElement paragraph, string tagSymbol, string tagName, int tagType = 0, bool tagStatus = false)
```

Comment:
I'm moving this to PageEditor class. Trying to keep the Page class focused on page-level properties and definitions and let PageEditor modify the content of the page.